opencensus/doc.go

```diff
@@ -1,2 +1,2 @@
 // Package opencensus for the Stackdriver Prometheus collector.
-package opencensus
\ No newline at end of file
+package opencensus
```

Comment:
Not a big deal, but this seems like an unrelated change…

Reply:
reverted out; my editor clearly got ahead of itself